worlds/hk/docs/setup_pt_br.md

```diff
@@ -0,0 +1,52 @@
+# Guia de configuração para Hollow Knight no Archipelago
+
+## Programas obrigatórios
+* Baixe e extraia o Lumafly Mod Manager (gerenciador de mods Lumafly) do [Site Lumafly](https://themulhima.github.io/Lumafly/).
+* Uma cópia legal de Hollow Knight.
+   * Versões Steam, Gog, e Xbox Game Pass do jogo são suportadas.
+   * Windows, Mac, e Linux (incluindo Steam Deck) são suportados.
+
+## Instalando o mod Archipelago Mod usando Lumafly
+1. Abra o Lumafly e confirme que ele localizou sua pasta de instalação do hollow knight.
+2. Clique em "Install (instalar)" perto da opção "Archipelago" mod.
+   * Se quiser, instale também o "Archipelago Map Mod (mod do mapa do archipelago)" para usá-lo como rastreador dentro do jogo.
+3. Abra o jogo, tudo preparado!
+
+### O que fazer se o Lumafly falha em encontrar a sua pasta de instalação
+1. Encontre a pasta manualmente.
+   * Xbox Game Pass:
+      1. Entre no seu aplicativo XBox e mova seu mouse em cima de "Hollow Knight" na sua barra da esquerda. 
+      2. Clique nos 3 pontos depois clique gerenciar.
+      3. Vá nos arquivos e selecione procurar. 
+      4. Clique em "Hollow Knight", depois em "Conteúdo", depois clique na barra com o endereço e a copie.
```

Comment:
Considering how you've been doing English labels with the translation in parenthesis, I think the same should apply to "Conteúdo" here, since in my testing the folder is called "Content"
![image](https://github.com/user-attachments/assets/b6860631-54c1-48cc-9b2e-6315828b3767)